changelog/12204.bugfix.rst

```diff
@@ -1,7 +1,11 @@
-Fix a regression in pytest 8.0 where tracebacks get longer and longer when multiple tests fail due to a shared higher-scope fixture which raised.
+Fixed a regression in pytest 8.0 where tracebacks get longer and longer when multiple
+tests fail due to a shared higher-scope fixture which raised -- by :user:`bluetech`.
 
-Also fix a similar regression in pytest 5.4 for collectors which raise during setup.
+Also fixed a similar regression in pytest 5.4 for collectors which raise during setup.
 
 The fix necessitated internal changes which may affect some plugins:
-- ``FixtureDef.cached_result[2]`` is now a tuple ``(exc, tb)`` instead of ``exc``.
-- ``SetupState.stack`` failures are now a tuple ``(exc, tb)`` instead of ``exc``.
+
+* ``FixtureDef.cached_result[2]`` is now a tuple ``(exc, tb)``
+  instead of ``exc``.
+* ``SetupState.stack`` failures are now a tuple ``(exc, tb)``
+  instead of ``exc``.
```

Comment:
Note: It took me a while to understand why this wasn't being displayed as a list in the preview. I inspected the HTML and found out that with this, the markup correctly uses `<li>`. Then, I saw that the bullets were disabled by inline CSS. And finally, I found the root cause to be https://github.com/pytest-dev/pytest/pull/12326/files#r1662635927.

Nevertheless, the RST markup fix in this PR is correct. The new cosmetic problem is elsewhere, in the theme patches.